tests/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
```

Comment:
I would prefer if these were tests under `proxmox-api/tests` in the way Cargo [recommends running integration tests](https://doc.rust-lang.org/rust-by-example/testing/integration_testing.html)

Reply:
all right, I ll delete the tests crate